manifests/infra/redis-operator/base/role_binding.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding-redis-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

公式も`cluster-admin`になっていますが、これじゃないんですかねー
```suggestion
  name: manager-role-redis-operator
```